### ISSUE TITLE
Use an Upstream base image

### ISSUE THREAD

**Issue by KTanAug21** [fly-apps/dockerfile-laravel#42]
**WHAT AND WHY**
Update the generated Dockerfile to use an upstream image, similar to how the current [laravel-docker repository is.](https://github.com/fly-apps/laravel-docker/blob/main/src/Dockerfile). 

There are three primary reasons behind this move:
1. To remove the scope of having to maintain the custom image built from the laravel-docker repository for SOC2 compliance. This was brought up from a customer request for periodic security patches and maintenance for the custom image we have provided, which we currently cannot fully commit to.
2. Consistency: The generated Dockerfiles for  other frameworks like Rails, Elixir, Javascript, all use upstream images as base. For consistency, we should also follow for the Dockerfile generated for Laravel apps.
3. Finally, this move will result in depositing config files to the users' project directory where they can visibly and readily find them, providing them full control over these files, as I quote @rubys : 
> they can rerun dockerfile-laravel when they want to pick up changes, hold back, and even make their own changes to these files."

___________________________________
**HOW:**
It's simple really! The laravel-docker repository already makes use of a Dockerfile that makes use of an upstream base image. So, we'll just make use of that Dockerfile. Further, since _that_ Dockerfile requires several config file, we'll copy those over to the dockerfile-laravel repository as well.

Basically copy over the laravel-docker logic to dockerfile-laravel. This will result in the same Dockerfile logic, while using an upstream base image instead of the custom one we've provided.

_______________________________________
**NOTES:**
1. For reference [here is the discourse post ](https://flyio.discourse.team/t/how-to-approach-securtiy-patches-for-fly-apps-laravel-docker/6484/4)tackling this issue 
2.  To make this a clean move and reduce the number of files to check. I'll create more than one PR to address this issue: 
    a. Move the config files required by the laravel-dockerfile over to this repository
    b. Move the Dockerfile logic of laravel-dockerfile over to this repository
